### PR TITLE
Notify admin with inititiatives content when a user destroys his account

### PR DESCRIPTION
**Pull request by armandfardeau**

### CODE REVIEW

**Review by Quentinchampenois — APPROVED** [review]
Seems good to me !

File: Gemfile.lock [hunk 1]
```diff
@@ -928,7 +928,7 @@ DEPENDENCIES
   wkhtmltopdf-binary
 
 RUBY VERSION
-   ruby 2.6.5p114
+   ruby 2.7.1p83
```
Comment (armandfardeau) [inline Gemfile.lock]:
```suggestion
   ruby 2.6.5p114
```

File: Gemfile.lock [hunk 2]
```diff
@@ -928,7 +928,7 @@ DEPENDENCIES
...
 
 BUNDLED WITH
-   2.2.16
+   2.2.24
```
Comment (armandfardeau) [inline Gemfile.lock]:
```suggestion
   2.2.16
```

File: .tool-versions [hunk 3]
```diff
@@ -0,0 +1 @@
+ruby 2.7.1
```
Comment (armandfardeau) [inline .tool-versions]:
```suggestion
ruby 2.6.5
```

**Comment by paulinebessoles** [comment]
Thanks @armandfardeau ! This a ok for me, I just have one question: would it be possible to not send the mail at all when no initiatives are displayed in it?
Precisely in those cases:

- Account with no initiatives
- Account with initiatives with excluded states (created, discarded, classified)
- Account with initiatives on which the user is co-author

If it's not possible or too complicated, we're good to send this PR and you can merge it :)

**Review by paulinebessoles — APPROVED** [review]

**Pull request merged**